src/nfa/nfa.cc

```diff
@@ -540,20 +540,60 @@ void Nfa::get_one_letter_aut(Nfa& result) const {
     result = get_one_letter_aut();
 }
 
-StateSet Nfa::post(const StateSet& states, const Symbol& symbol) const {
+StateSet Nfa::post(const StateSet& states, const Symbol& symbol, const EpsilonClosureOpt eps_closure_opt) const {
```

Comment:
```suggestion
StateSet Nfa::post(const StateSet& states, const Symbol& symbol, const EpsilonClosureOpt epsilon_closure_opt) const {
```
I would use the longer names, especially if they are only in the local function and as parameters. I am tired of these `prfx`, `lang`, `eps`, `tgt`, `src`, `symb`, etc. I really think this makes the code more readable.

Furthermore, I am not sure whether `opt` is the best name for additional parameters, but “opt” is a verb for “to choose” or similar, so it might be quite a nice approach. Maybe switching the prefix and suffix like `opt_epsilon_closure`? To make it clear that it is something to opt-in on?

Reply:
After some thinking, I'll change `opt` to `flags` and `eps` to `epsilon`.

Reply:
Maybe wait. I really like opt when I think about it. And it is shorter.

Reply:
I meant wait with the opt to flags change. I would change the `eps`. But that is just a nitpick now.